### PR TITLE
Set gateway on start-up

### PR DESCRIPTION
**Pull request by alistair-singh**
Polkadot-sdk: https://github.com/Snowfork/polkadot-sdk/pull/66

### CODE REVIEW

File: web/packages/test/scripts/configure-bridgehub.sh [hunk 1]
```diff
@@ -69,7 +69,17 @@ open_hrmp_channels()
     open_hrmp_channel "${relaychain_ws_url}" "${relaychain_sudo_seed}" 2000 1000 8 512 # Assethub -> Penpal
 }
 
+set_gateway()
+{
+    echo "Setting gateway contract"
+    local storage_key=$(echo $GATEWAY_STORAGE_KEY | cut -c3-)
+    local gateway=$(echo $GATEWAY_PROXY_CONTRACT | cut -c3-)
+    local transact_call="0x00040440"$storage_key"50"$gateway
```
Comment (yrong) [inline web/packages/test/scripts/configure-bridgehub.sh:77]: Cool!  I would assume it's a `System::set_storage()` call here, Just curious how to generate the `storage_key` here.

Comment (alistair-singh) [reply web/packages/test/scripts/configure-bridgehub.sh:77]: So basically I modified this unit test to print it out. The generation of the key looks stable.

https://github.com/Snowfork/polkadot-sdk/pull/66/files#diff-79a28d4d3e1749050341c2424f00c4c139825b1a20937767f83e58b95166735cR153

The generation of this key is stable across all runtimes:
https://github.com/Snowfork/polkadot-sdk/blob/alistair/gateway-in-storage/substrate/frame/support/src/lib.rs#L333

**Review by vgeddes — APPROVED** [review]
+1

File: smoketest/tests/send_token_to_penpal.rs [hunk 2]
```diff
@@ -68,6 +68,7 @@ async fn send_token_to_penpal() {
 			weth.address(),
 			PENPAL_PARA_ID,
 			i_gateway::MultiAddress { kind: 1, data: FERDIE.into() },
+			4_000_000_000,
```
Comment (claravanstaden) [inline smoketest/tests/send_token_to_penpal.rs:71]: Nice catch!

**Review by claravanstaden — APPROVED** [review]

**Pull request merged**